azure-devops/azext_devops/dev/team/wiki.py

```diff
@@ -177,7 +177,7 @@ def update_page(wiki, path, version, comment=_DEFAULT_PAGE_UPDATE_MESSAGE, conte
                                              project=project, path=path, version=version, comment=comment)
 
 
-def get_page(wiki, path, version=None, recursion_level=None, open=False,  # pylint: disable=redefined-builtin
```

Comment:
Removing this since it is not required in get page. We can discuss on the need of getting multiple pages through a command and its structure.